released/packages/coq-coqeal/coq-coqeal.dev/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+synopsis: "CoqEAL - The Coq Effective Algebra Library"
+description: """\
```

Comment:
Spurious backslash?

Reply:
These are systematically inserted by `opam-publish`, I did not pay attention in the first place. I am wondering why it does that...